### DISCUSSION TITLE
Update PowerShell Homebrew Tap Formula to v7.4.2?

### DISCUSSION THREAD

**Discussion by EricZBA** [PowerShell/PowerShell#21615] [Q&A] [answered]
How do we get [pwshBot](https://github.com/PowerShell/Homebrew-Tap/commits?author=pwshBot) to update the PowerShell Stable [Homebrew-Tap Formula](https://github.com/PowerShell/Homebrew-Tap/blob/master/Formula/powershell.rb) to v7.4.2? Does someone upstream need to trigger that manually, or should it happen automatically?

The last commit is from January when it was updated to version 7.4.1 -> https://github.com/PowerShell/Homebrew-Tap/commit/15478afbb394f9eced8ecc7674a1104a5d862f95

Thank you.

**Comment by rhubarb-geek-nz** [comment]
> How do I remove Poweshell as being the default Terminal application?

If you are referring to the Windows Terminal then click on the down arrow in the title bar, choose "Settings" and choose a "Default profile", eg mine is "Command Prompt"

**Comment by EricZBA** [comment, marked as answer]
Issue has been resolved by https://github.com/PowerShell/Homebrew-Tap/pull/1246

Thank you @StevenBucher98 @TravisEz13
